products/opensuse/main.pm

```diff
@@ -401,15 +401,6 @@ else {
         loadtest "remote/remote_target";
         load_reboot_tests();
     }
-    elsif (is_jeos) {
```

Comment:
Why is this no longer required?

Reply:
Looks like it was not used since there is [another branch](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/77f042ffa03ded9cdba3aaa641e84756757d3991/products/opensuse/main.pm#L277). It is handling registration using non-existing module `sccreg`.